Formula/getxbook.rb

```diff
@@ -5,6 +5,11 @@ class Getxbook < Formula
   sha256 "7a4b1636ecb6dace814b818d9ff6a68167799b81ac6fc4dca1485efd48cf1c46"
```

Comment:
```suggestion
  sha256 "7a4b1636ecb6dace814b818d9ff6a68167799b81ac6fc4dca1485efd48cf1c46"
  license "ISC"
```
The website indicates that the license used is the `ISC License`.

Reply:
So it does. Thanks, @JCount!